### DISCUSSION TITLE
Lecture 4:Question about .env file

### DISCUSSION THREAD

**Discussion by Dahe-Pan** [smartcontractkit/full-blockchain-solidity-course-py#1859] [Q&A] [answered]
I follow the step to create one .env file inside web3 folder, and write "export variable=value", but unlike in the video, the 'export' in my file is still white and when I use getenv function to use it from another .py file, it returns 'None', when I add new environmental variables from Advanced system setting, it worked. Can anyone explain how to set .env file? I'm very thankful.

**Comment by Niferu** [comment, marked as answer]
Hello @Dahe-Pan 

Please show us your code where you are importing .env and **example** .env file you are using.

**Reply by Dahe-Pan** [reply]
Thanks for your help, I just found the mistake I made which is I forgot to use dotenv.

**Reply by Niferu** [reply]
Yup I thought that might be the case too, glad it is fixed now :)

**Reply by cromewar** [reply]
Thanks @Neftyr, closing this.
